src/convertToObject.js

```diff
@@ -6,7 +6,23 @@
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  const sourceArray = sourceString.split(';');
+
+  const convertsObject = {};
+
+  sourceArray.forEach((style) => {
+    const firstKeyIndex = style.indexOf(':');
+
+    if (firstKeyIndex !== -1) {
+      const key = style.slice(0, firstKeyIndex).trim();
+      const firstValueIndex = firstKeyIndex + 1;
+      const value = style.slice(firstValueIndex).trim();
+
+      convertsObject[key] = value;
+    }
+  });
```

Comment:
[CODE PRACTICE] - Instead of using `forEach` and mutating an object, consider using `reduce` to transform the array into an object. This aligns with functional programming practices and avoids side-effects.